frontend/src/metabase/dashboard/components/EmbedMenu/AdminEmbedMenu.tsx

```diff
@@ -30,7 +30,9 @@ export const AdminEmbedMenu = ({
     getSetting(state, "enable-embedding"),
   );
 
-  const target = <DashboardEmbedHeaderButton />;
+  const target = (
+    <DashboardEmbedHeaderButton hasBackground={resourceType === "dashboard"} />
```

Comment:
For now I am just using a boolean here for `hasBackground` - there's definitely something to be said for using different target components but I think for now this solution works and would just be nice to have in master until I can take some time to split everything out next year